parsers/myrddin.c

```diff
@@ -0,0 +1,46 @@
+/*
+ *   Copyright (c) 2016, Ori Bernstein
+ *
+ *   This source code is released for free distribution under the terms of the
+ *   GNU General Public License.
+ *
+ *   This module contains functions for generating tags for Myrddin language
+ *   files.
+ */
+
+/*
+ *   INCLUDE FILES
+ */
+#include "general.h"  /* must always come first */
+#include <string.h>
+#include "parse.h"
+
+static const tagRegexTable myrddinTagRegexTable [] = {
+	{"^([ \t]*extern)?[ \t]*const[ \t]+([a-zA-Z_][a-zA-Z0-9_]*)",
+		"\\2", "f,function,functions", NULL},
+	{"^([ \t]*extern)?[ \t]*var[ \t]+([a-zA-Z_][a-zA-Z0-9_]*)",
+		"\\2", "v,var,variables", NULL},
+	{"^[ \t]*type[ \t]+([a-zA-Z_][a-zA-Z0-9_]*)[ \t]*=",
+		"\\1", "t,ty,type", NULL},
```

Comment:
`ty` is used as long kind name. However, I think, `type` is better. I get an expression that `ty` is too short .

How about
  `t,type,types`
instead ?